src/pages/iou/IOUModal.js

```diff
@@ -140,6 +141,11 @@ class IOUModal extends Component {
             Navigation.dismissModal();
         }
 
+        // If transaction fails, handling it here
+        if (prevProps.iou.creatingIOUTransaction && this.props.iou.error === true) {
+            this.handleTransactionError();
```

Comment:
Since the method is just a single line, you can just use `  this.setState({currentStepIndex: 0});` here

Reply:
It show a warning in lint,
<img width="804" alt="Screenshot 2021-08-05 at 1 04 49 AM" src="https://user-images.githubusercontent.com/85645967/128243929-b704d520-4517-4b3a-b7a5-eb733e8b64be.png">

So need to a disable lint comment,
<img width="754" alt="Screenshot 2021-08-05 at 1 03 58 AM" src="https://user-images.githubusercontent.com/85645967/128243774-a0688c64-5bf6-4881-8891-e1afd9478e44.png">

Can we keep the current changes are just check in new changes? @thienlnam

Reply:
Putting it in a method just is a workaround for the lint issue instead of actually solving the problem.  I think you might be able to solve it with `componentDidUpdate(prevProps, currentProps)` and use `currentProps` instead of `this.props` in `componentDidUpdate`

Reply:
<img width="990" alt="Screenshot 2021-08-05 at 3 14 44 AM" src="https://user-images.githubusercontent.com/85645967/128259238-2f34cfd1-9733-4ea3-8aba-d8868ba13f25.png">

Reply:
Ah my bad 🤦  did not look at how that worked - going to test this to make sure no infinite loops are occuring